io/net/vlan_test.py

```diff
@@ -53,13 +55,23 @@ def setUp(self):
         self.get_ips()
 
     def parameters(self):
+        local = LocalHost()
+        interfaces = os.listdir('/sys/class/net')
+        device = self.params.get("interface", '*', default=None)
```

Comment:
can we keep the default same across the test either `None` or `""`?
any reason for adding * here?

Reply:
thats the path parameter, not required can be removed